cdhweb/pages/templates/cdhpages/blocks/download_block.html

```diff
@@ -9,11 +9,17 @@ <h2>{{ self.heading }}</h2>
     <p>{{ self.description }}</p>
   {% endif %}
 
-  <ul class="downloads__list">
+  <ul>
     {% for file in file_links %}
       <li>
-        <a {% if file.file.url %}href="{{ file.file.url }}"{% endif %} class="downloads__link" download>
-          {{ file.title }} ({{ file.file_size | filesizeformat }} {{ file.file_type }})
+        <a href="{{ file.file.url }}" download>
```

Comment:
FYI @sarahframe I removed the href conditional here, as I don't see how someone could add a download link without one. That OK?